### ISSUE TITLE
Can not tunnel:open

### ISSUE THREAD

**Issue by salines** [platformsh/platformsh-cli#774]
```
platform tunnel:open
Enter a number to choose a project:
  [0] ****** (******)
  [1] ********* (******)
 > 1
```
 
exit without any message

**Comment by pjcdawkins** [comment]
Hi, could you re-run the command with `-v`? It would show up any PHP errors.

**Comment by cferthorney** [comment]
I'm having identical issues. Currently running `3.40.4` PHP 7.3.2 using standard Homebrew configuration:
```
brew info php@7.3
php: stable 7.3.2 (bottled)
General-purpose scripting language
https://secure.php.net/
/usr/local/Cellar/php/7.3.2 (521 files, 77MB) *
  Poured from bottle on 2019-02-08 at 10:59:54
From: https://github.com/Homebrew/homebrew-core/blob/master/Formula/php.rb
==> Dependencies
Build: httpd ✘, pkg-config ✔
Required: apr ✔, apr-util ✔, argon2 ✔, aspell ✔, autoconf ✔, curl-openssl ✔, freetds ✔, freetype ✔, gettext ✔, glib ✔, gmp ✔, icu4c ✔, jpeg ✔, libpng ✔, libpq ✔, libsodium ✔, libzip ✔, openldap ✔, openssl ✔, sqlite ✔, tidy-html5 ✔, unixodbc ✔, webp ✔
==> Caveats
To enable PHP in Apache add the following to httpd.conf and restart Apache:
    LoadModule php7_module /usr/local/opt/php/lib/httpd/modules/libphp7.so

    <FilesMatch \.php$>
        SetHandler application/x-httpd-php
    </FilesMatch>

Finally, check DirectoryIndex includes index.php
    DirectoryIndex index.php index.html

The php.ini and php-fpm.ini file can be found in:
    /usr/local/etc/php/7.3/

To have launchd start php now and restart at login:
  brew services start php
Or, if you don't want/need a background service you can just run:
  php-fpm
==> Analytics
install: 33,993 (30 days), 109,962 (90 days), 457,168 (365 days)
install_on_request: 30,608 (30 days), 99,245 (90 days), 412,373 (365 days)
build_error: 0 (30 days)
```

I have xdebug and imagick installed, but uninstalling them seems to make no difference to the results.


```
user@machine ~/Development/{client}/Drupal/{project}/docroot (master) $ platform -v tunnels
No tunnels found.
user@machine ~/Development/{client}/Drupal/{project}/docroot (master) $ platform -v tunnel:open
Running command: 'git' 'rev-parse' '--abbrev-ref' '--symbolic-full-name' '@{u}'
Running command: 'command' '-v' 'drush'
Running command: '/usr/local/bin/drush' 'version' '--format=string'
Running command: '/usr/local/bin/drush' '@none' 'sa' '--format=json' '@{ENV ID}'
Are you sure you want to open SSH tunnel(s) to the master (production) environment? [y/N] y

user@machine ~/Development/{client}/Drupal/{project}/docroot (master) $ platform -v tunnels
No tunnels found.
```
My SSH key is  stored in a session via `ssh-add`.

Let me know if there is any further information you need

**Comment by pjcdawkins** [comment]
Reproduced with PHP 7.3, thank you @cferthorney

It's failing silently on this regular expression:

https://github.com/platformsh/platformsh-cli/blob/503cb7301afffdae8700e5e59170c81e89c6c935/src/Command/Tunnel/TunnelCommandBase.php#L206

**Comment by pjcdawkins** [comment]
Thanks @cferthorney and @salines for the reports.

This should now be resolved in version 3.40.5, which is available now via `platform up`. That version includes a workaround for the PHP bug, by disabling `pcre.jit` for commands starting with `platform tunnel:open` (or `platform t:o`).

Please check if you still have a silent failure or segfault with `tunnel:open`.

**Comment by pjcdawkins** [comment]
The original bug was fixed in PHP since version 7.3.11:
https://github.com/php/php-src/commit/ab61d5caf926e080fb5729dbef9ae091169cec88

So I'll remove this workaround via issue #1092
